bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckablePullRequest.java

```diff
@@ -192,6 +225,8 @@ void executeChecks(Hash localHash, CensusInstance censusInstance, PullRequestChe
         if (conf.isEmpty()) {
             throw new RuntimeException("Failed to parse jcheck configuration at: " + targetHash() + " with extra: " + additionalConfiguration);
         }
+        // Set the review requirements for using in CheckRun#getChecksList
+        setReviewRequirements(conf.get());
```

Comment:
I don't think it is good to put such code in `CheckablePullRequest#executeChecks`. But I can't find a better way to do that because we need the complete `JCheckConfiguration` config information.

An alternative way is duplicating the following code in the `CheckRun` so that we can get the complete config information.

```
        Optional<JCheckConfiguration> conf;
        if (confOverride != null) {
            conf = JCheck.parseConfiguration(confOverride, additionalConfiguration);
        } else {
            conf = JCheck.parseConfiguration(localRepo, targetHash(), additionalConfiguration);
        }
        if (conf.isEmpty()) {
            throw new RuntimeException("Failed to parse jcheck configuration at: " + targetHash() + " with extra: " + additionalConfiguration);
        }
```